app/presenters/concerns/displays_image.rb

```diff
@@ -3,8 +3,11 @@
 module DisplaysImage
   extend ActiveSupport::Concern
 
+  # Creates a display image only where FileSet is an image.
+  #
+  # @return [IIIFManifest::DisplayImage] the display image required by the manifest builder.
   def display_image
-    return nil unless FileSet.exists?(id)
+    return nil unless FileSet.exists?(id) && FileSet.find(id).image?
```

Comment:
Ah, I wondered about that - it was just so neat this way! I can do something with solr. Will work on it once back from hols next week.